### PR TITLE
zh-CN: translate tutorial/image/cnn.ipynb

### PR DESCRIPTION
**Pull request by Hedgeh09**
Translate "tutorial/image/cnn.ipynb" into Simplified Chinese.

### CODE REVIEW

File: site/zh-cn/tutorials/images/cnn.ipynb [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "jRFxccghyMVo"
+      },
+      "source": [
+        "### 下载并准备 CIFAR10 数据集\n",
+        "\n",
+        "\n",
+        "CIFAR10 数据集包含10类，共60000张彩色图片，每类图片有6000张。此数据集中50000个样例被作为训练集，剩余10000个样例作为测试集。类之间相互度立，不存在重叠的部分。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/images/cnn.ipynb:117]:
```suggestion
        "CIFAR10 数据集包含 10 类，共 60000 张彩色图片，每类图片有 6000 张。此数据集中 50000 个样例被作为训练集，剩余 10000 个样例作为测试集。类之间相互度立，不存在重叠的部分。"
```

File: site/zh-cn/tutorials/images/cnn.ipynb [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "7wArwCTJJlUa"
+      },
+      "source": [
+        "### 验证数据\n",
+        "\n",
+        "我们将测试集的前25张图片和类名打印出来，来确保数据集被正确加载。\n"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/images/cnn.ipynb:146]:
```suggestion
        "我们将测试集的前 25 张图片和类名打印出来，来确保数据集被正确加载。\n"
```

File: site/zh-cn/tutorials/images/cnn.ipynb [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "3hQvqXpNyN3x"
+      },
+      "source": [
+        "下方展示的6行代码声明了了一个常见卷积神经网络，由几个 [Conv2D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D) 和 [MaxPooling2D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/MaxPool2D) 层组成。\n",
+        "\n",
+        "CNN 的输入是张量 (Tensor) 形式的 (image_height, image_width, color_channels)，包含了图像高度，宽度，及颜色信息。不需要输入 batch size。如果您不熟悉图像处理，颜色信息建议您使用 RGB 色彩模式，此模式下，`color_channels` 为 `(R,G,B)` 分别对应 RGB 的三个颜色通道（color channel）。在此示例中，我们的 CNN 输入，CIFAR 数据集中的图片，形状是 `(32, 32, 3)`。您可以在声明第一层时将形状赋值给参数 `input_shape` 。\n"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/images/cnn.ipynb:194]:
```suggestion
        "CNN 的输入是张量 (Tensor) 形式的 (image_height, image_width, color_channels)，包含了图像高度、宽度及颜色信息。不需要输入 batch size。如果您不熟悉图像处理，颜色信息建议您使用 RGB 色彩模式，此模式下，`color_channels` 为 `(R,G,B)` 分别对应 RGB 的三个颜色通道（color channel）。在此示例中，我们的 CNN 输入，CIFAR 数据集中的图片，形状是 `(32, 32, 3)`。您可以在声明第一层时将形状赋值给参数 `input_shape` 。\n"
```

File: site/zh-cn/tutorials/images/cnn.ipynb [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "_j-AXYeZ2GO5"
+      },
+      "source": [
+        "在上面的结构中，您可以看到每个 Conv2D 和 MaxPooling2D 层的输出都是一个3维的张量 (Tensor)，其形状描述了 (height, width, channels)。越深的层中，宽度和高度都会收缩。每个 Conv2D 层输出的通道数量 (channels) 取决于声明层时的第一个参数（如：上面代码中的32或64）。这样，由于宽度和高度的收缩，您便可以（从运算的角度）增加每个 Conv2D 层输出的通道数量 (channels)。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/images/cnn.ipynb:247]:
```suggestion
        "在上面的结构中，您可以看到每个 Conv2D 和 MaxPooling2D 层的输出都是一个三维的张量 (Tensor)，其形状描述了 (height, width, channels)。越深的层中，宽度和高度都会收缩。每个 Conv2D 层输出的通道数量 (channels) 取决于声明层时的第一个参数（如：上面代码中的32或64）。这样，由于宽度和高度的收缩，您便可以（从运算的角度）增加每个 Conv2D 层输出的通道数量 (channels)。"
```

**Comment by Hedgeh09** [comment]
@Mr-Linus Format has been modified.

File: site/zh-cn/tutorials/images/cnn.ipynb [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/tutorials/estimator/boosted_trees\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorFlow.google.cn 上查看</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/tutorials/estimator/boosted_trees.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/tutorials/estimator/boosted_trees.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\">在 GitHub 上查看源代码</a>\n",
+        "  </td>\n",
```
Comment (lamberta) [inline site/zh-cn/tutorials/images/cnn.ipynb:64]: Thanks. Can you please update these button URLs to point to this file?

Comment (Hedgeh09) [reply site/zh-cn/tutorials/images/cnn.ipynb]: Thanks for pointing them out, links have been fixed.

**Comment by Hedgeh09** [comment]
@duanw0916 @lamberta Please check the problem below.

This translated file is not shown in the official website, it should be shown on: https://tensorflow.google.cn/tutorials/images/cnn

**Comment by lamberta** [comment]
Hi @Hedgeh09 ,
This one may not be running because it has a bad kernelspec name:

```
"kernelspec": {
      "display_name": "tf",
      "language": "python",
      "name": "tf"
    }
```
It should be:
```
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    }
```
Can you try updating that? Thanks

**Comment by lamberta** [comment]
Thanks for check back on this.
I updated `nbfmt` to handle bad kernelspec names. This notebook will be fixed in https://github.com/tensorflow/docs-l10n/pull/280/files#diff-9925f36c6d903ee096e826f648d688a9

**Comment by Hedgeh09** [comment]
> Hi @Hedgeh09 ,
> This one may not be running because it has a bad kernelspec name:
> 
> ```
> "kernelspec": {
>       "display_name": "tf",
>       "language": "python",
>       "name": "tf"
>     }
> ```
> 
> It should be:
> 
> ```
>     "kernelspec": {
>       "display_name": "Python 3",
>       "language": "python",
>       "name": "python3"
>     }
> ```
> 
> Can you try updating that? Thanks

Thanks for fixing those, I didn't get notification because that time was midnight in my area hhhhh.

**Pull request merged**